Source/JavaScriptCore/bytecode/InlineCacheCompiler.cpp

```diff
@@ -4887,8 +4887,18 @@ static MacroAssemblerCodeRef<JITThunkPtrTag> getByIdLoadHandlerCodeGenerator(VM&
     return FINALIZE_THUNK(patchBuffer, JITThunkPtrTag, "GetById Load handler"_s, "GetById Load handler");
 }
 
+MacroAssemblerCodeRef<JITThunkPtrTag> getByIdLoadOwnPropertyHandlerCodeGenerator(VM& vm)
+{
+    return getByIdLoadHandlerCodeGeneratorImpl<true>(vm);
```

Comment:
This reminds me that these bools should probably be enums. Can we change that while you're here?

Reply:
I don't think using enum is better in particular here since it is only used in the function above, and scattering many enums for that does not make it readable (since it is wrapped with a function anyway, it does not have much improvement for readability). I would like to just use comment here.